pkg/util/nsenter/nsenter.go

```diff
@@ -83,20 +83,23 @@ func NewNsenter() *Nsenter {
 	}
 	// search for the required commands in other locations besides /usr/bin
 	for binary := range ne.paths {
-		// default to root
-		ne.paths[binary] = filepath.Join("/", binary)
+		found := false
 		for _, path := range []string{"/bin", "/usr/sbin", "/usr/bin"} {
```

Comment:
to retain support for /<binary> you might consider adding `"/"` to the list of searched paths